packages/typegpu/src/types.ts

```diff
@@ -114,7 +115,7 @@ export interface TgpuBindable<
   TData extends AnyTgpuData = AnyTgpuData,
   TUsage extends BufferUsage = BufferUsage,
 > extends TgpuResolvable {
-  readonly allocatable: TgpuAllocatable<TData>;
+  readonly allocatable: TgpuBuffer<TData>;
```

Comment:
is `TgpuAllocatable` interface still useful? can't remember what other types of allocatables we were planning to introduce